### PR TITLE
Update guestbook-logs-metrics-with-elk.md

### PR DESCRIPTION
**Pull request by atomerz**
The description was a little confusing. Moved around some text and removed duplicated/unnecessary parts.

### CODE REVIEW

**Comment by atomerz** [comment]
CLA signed.

File: content/en/docs/tutorials/stateless-application/guestbook-logs-metrics-with-elk.md [hunk 1]
```diff
@@ -60,23 +60,15 @@ kubectl create clusterrolebinding cluster-admin-binding \
...
 git clone https://github.com/kubernetes/kube-state-metrics.git kube-state-metrics
 kubectl apply -f kube-state-metrics/examples/standard
-kubectl get pods --namespace=kube-system | grep kube-state-metrics
 ```
-Verify that kube-state-metrics is running and ready
+
+### Check to see if kube-state-metrics is running
 ```shell
-kubectl get pods -n kube-system -l app.kubernetes.io/name=kube-state-metrics
+kubectl get pods --namespace=kube-system | grep kube-state
```
Comment (sftim) [inline content/en/docs/tutorials/stateless-application/guestbook-logs-metrics-with-elk.md:70]: Would
```suggestion
kubectl get pods --namespace=kube-system -l app.kubernetes.io/name=kube-state-metrics
```
also work? (If you're on Windows, for example, you might not have `grep`).

**Review by sftim — COMMENTED** [review]
/lgtm

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-24071--kubernetes-io-master-staging.netlify.app/docs/tutorials/stateless-application/guestbook-logs-metrics-with-elk/#install-kube-state-metrics

Thanks @atomerz .
/lgtm

**Pull request merged**